src/Microsoft.AspNet.OData.Shared/GetNextPageHelper.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.AspNet.OData
     /// </summary>
     internal static partial class GetNextPageHelper
     {
-        internal static Uri GetNextPageLink(Uri requestUri, IEnumerable<KeyValuePair<string, string>> queryParameters, int pageSize)
+        internal static Uri GetNextPageLink(Uri requestUri, IEnumerable<KeyValuePair<string, string>> queryParameters, int pageSize, string skipTokenValue = "")
```

Comment:
>= "" [](start = 155, length = 4)

why can't use 'null"?

Reply:
+1

---
In reply to: [238494040](https://github.com/OData/WebApi/pull/1689#discussion_r238494040) [](ancestors = 238494040)